openslides/core/static/js/core/site.js

```diff
@@ -354,6 +360,32 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+// Options for TinyMCE editor used in various create and edit views.
+.factory('EditorOptions', [
+    'gettextCatalog',
+    'Mediafile',
```

Comment:
Hm. Sounds like a nice feature but it would be new and we should discuss this (because of its consequences for the app structure) when we have more time.